cli/args/flags.rs

```diff
@@ -2880,6 +2881,13 @@ update to a different location, use the --output flag
             .help("Upgrade to canary builds")
             .action(ArgAction::SetTrue),
         )
+        .arg(
+          Arg::new("release_candidate")
```

Comment:
```suggestion
          Arg::new("release-candidate")
```

other flags we have use `-` (like `dry-run`)